ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/launcher/FeatureBlock.java

```diff
@@ -391,7 +391,7 @@ private void handleValidate() {
 				if (fDialog == null) {
 					if (fOperation.hasErrors()) {
 						fDialog = new PluginStatusDialog(getShell(), SWT.MODELESS | SWT.CLOSE | SWT.BORDER | SWT.TITLE | SWT.RESIZE);
-						fDialog.setInput(fOperation.getInput());
+						fDialog.setInput(fOperation.getInput(), fOperation.getfLaunchConfiguration());
```

Comment:
Why is the new parameter needed, if the operation has a public getter for the config?

Reply:
To call the getter method in the PluginStatusDialog class, the instance of LaunchValidationOperation is needed. Therefore, I introduced a new parameter in setIput() to obtain the current launch configuration. Is there a better alternative way for this?